optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -510,15 +510,7 @@ def load_model(
 
         return encoder_session, decoder_session, decoder_with_past_session
 
-    def _save_pretrained(
-        self,
-        save_directory: Union[str, Path],
-        # TODO: should we make the default values available here?
-        encoder_file_name: str = ONNX_ENCODER_NAME,
-        decoder_file_name: str = ONNX_DECODER_NAME,
-        decoder_with_past_file_name: str = ONNX_DECODER_WITH_PAST_NAME,
```

Comment:
It wont cause any issue not to have this possibility anymore? For optimized and quantized models?
cc @echarlaix

Reply:
Yes looking back it was a quick and dirty fix to remove these but I'll add them back, it is breaking change. Although I am not sure why those args were introduced in the first place - in transformers, you can not specify a path to a `.bin`.

Reply:
Actually, no, this is in the `save_pretrained`, I don't see any point in keeping it other than backward compatibility.